test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -482,7 +480,7 @@ export default class PerfCompare extends Component {
                     const metricPropsJSON = await getBenchmarkMetricProps(benchmark);
                     if (metricPropsJSON) {
                         this.metricsProps[benchmark] = metricPropsJSON;
-                        metricProps = metricPropsJSON.curMetricName;
+                        metricProps = metricPropsJSON[curMetricName];
```

Comment:
There is no need to change this.

Reply:
This is a bug fix here. There's a difference in calling the keys in JSON objects by using `obj.var` and obj[var]. I have done a test here below:
1) `obj.var` would return the value behind the `JSONkey=var`
2) `obj[var]` would return the value that the `JSONkey= valueof(var)`.
3) In this case, `curMetricName` is a variable that could be `startup`/`footprint` etc, and we need to find the key that equals those `startup`/`footprint` values. So using `obj[var]` would be right. otherwise it would return `undefined` always.

<img width="1122" alt="Screen Shot 2020-04-08 at 4 19 13 PM" src="https://user-images.githubusercontent.com/43182493/78829793-fcc40380-79b4-11ea-9386-f6b263346ed1.png">

Reply:
yes, you are right. :)